src/Core/functions.php

```diff
@@ -249,6 +249,31 @@ function env($key, $default = null)
 
 }
 
+if (!function_exists('warn')) {
+    /**
+     * Throws a warning.
+     *
+     * @param string $message The warning message.
+     * @return void
+     */
+    function warn($message)
```

Comment:
Following should I change this to `warning()`?
https://github.com/cakephp/cakephp/pull/11319

Reply:
That would be great.